core.py

```diff
@@ -88,6 +101,8 @@ class StaSh(object):
     Main application class. It initialize and wires the components and provide
     utility interfaces to running scripts.
     """
+    
+    PY3 = PY3
```

Comment:
The `PY3` on the left side is a attribute of the `StaSh()` class and its instances, the right side is a module-level `PY3` constants.
Both have a reason to exist: Because we need to check for `PY3` before creating the `StaSh` class, the module level one exists. And because `if _stash.PY3:` is a simpler syntax than importing or defining a custom `PY3` variable in each command that one also exists.